x-pack/plugins/cases/public/components/custom_fields/toggle/get_eui_table_column.tsx

```diff
@@ -0,0 +1,62 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React from 'react';
+
+import type { EuiTableComputedColumnType } from '@elastic/eui';
+import { EuiSwitch } from '@elastic/eui';
+
+import type { CaseUI } from '../../../containers/types';
+import { CustomFieldTypes } from '../../../../common/types/domain';
+import { getEmptyTagValue } from '../../empty_value';
+
+export const getEuiTableColumn = ({
```

Comment:
Should we set a `width`? It is a toogle and the size will always be small.

Reply:
An aside regarding column widths and the potential craziness of the table as more columns are added...

I think we can work towards having the table scroll horizontally.  Otherwise, we will constantly be battling this.  
So perhaps setting columns to auto (with a max-width) and just let the table overflow.  There weren't a11y concerns with this approach (we would just need to make sure we provide appropriate keyboard access).  But I think finding magic widths here is going to be a losing battle.  

Should this be built into the EUI table? Perhaps.  Seems like the team there is +1 for this, but this obviously is beyond this PR's scope.  

Example gif (with an option to pin a column or columns).  
![Screenshot--2023-10-26--Table Designs - UI Updates](https://github.com/elastic/kibana/assets/3756330/25a06ce0-9664-46d4-a7cf-083cac0a2408)

Reply:
Thanks, @mdefazio for the input. I totally agree that is impossible to find a good balance with the widths. Horizontally scrolling seems the way to go. @adcoelho What do you think about contributing to the EUI table as a stretch goal after we are done with the feature?